pkg/disk/v1alpha1/spc_client.go

```diff
@@ -19,12 +19,12 @@ package v1alpha1
 import (
 	apis "github.com/openebs/maya/pkg/apis/openebs.io/ndm/v1alpha1"
 	"github.com/pkg/errors"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
nit: metav1 ?

Reply:
We are planning to remove disk packages in near future anyways updated the alias to metav1